qa/workunits/suites/mgr_mod_snap_schedule.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/bash
+TESTDIR=$(dirname $0)
+# run for 16 minutes
+RUN_FOR=16
+EXPECTED_SNAPS=15
+RNG_SEED=${RNG_SEED:-$(date +%s)}
+
+echo "TESTDIR=$TESTDIR"
+echo "RUN_FOR=$RUN_FOR"
+echo "EXPECTED_SNAPS=$EXPECTED_SNAPS"
+echo "RNG_SEED=$RNG_SEED"
+echo "PYTHON=$(which python3)"
+
+ceph mgr module enable snap_schedule
+sleep 2
```

Comment:
what is the function of these sleeps?

Reply:
> what is the function of these sleeps?

these sleeps are just some time buffer around commands so that the system gets enough time to stabilize after executing the commands